dwave/system/samplers/dwave_sampler.py

```diff
@@ -252,15 +229,16 @@ def sample_ising(self, h, J, **kwargs):
 
             **kwargs:
                 Optional keyword arguments for the sampling method, specified per solver in
-                :attr:`.DWaveSampler.parameters`
+                :attr:`.DWaveSampler.parameters`.
+                `D-Wave System Documentation <https://docs.dwavesys.com/docs/latest/doc_solver_ref.html>`_
+                describes the parameters and properties supported on the D-Wave system.
 
         Returns:
             :class:`dimod.SampleSet`: A `dimod` :obj:`~dimod.SampleSet` object.
```

Comment:
Can we add some info here to help address #173 ?

Reply:
It has a link to dimod's sampleset class, and over there I'll address #173 by adding a link to https://docs.dwavesys.com/docs/latest/doc_timing.html (I'd rather not link to a lower level in the code, I can see the possibility that Measuring Computation Time undergoes some restructuring for the new architecture). This fits with @vgoliber's use case, "I click on SampleSet" and seems likely to me to be the general use case where most users will want all the information about the returned object rather than that particular part.

Reply:
I don't think SampleSet should contain information specific to any sampler. I understand that users click on SampleSet but that would be like looking at python's dict to understand embedding. Because it is the DWaveSampler that adds that information to SampleSet I think it needs to be captured here.

Reply:
That's a good point, I agree